Pipfile

```diff
@@ -24,6 +24,8 @@ html2text = "*"
 googletrans = "*"
 html5lib = "*"
 py-stackexchange = "*"
+pre-commit = "*"
```

Comment:
Unacceptable, pre-commit is in `[dev-packages]` there is absolutely no reason to include it in `[packages]`